docs/operator-manual/upgrading/2.4-2.5.md

```diff
@@ -97,14 +99,14 @@ When using `argocd app diff --local`, code from the repo server is run on the us
 
 In order to support CMPs and reduce local requirements, we have implemented *server-side generation* of local manifests via the `--server-side-generate` argument. For example, `argocd app diff --local repoDir --server-side-generate` will upload the contents of `repoDir` to the repo server and run your manifest generation pipeline against it, the same as it would for a Git repo.
 
-In v2.6, the `--server-side-generate` argument will become the default and client-side generation will be removed.
+In ~~v2.6~~ v2.7, the `--server-side-generate` argument will become the default and client-side generation will be removed.
```

Comment:
isn't it too soon to remove the client-side support in v2.7? Maybe we should have an option to have client-support available with a flag while making server-side default? so that users can fall back if they see any issue with server-side generation.

Reply:
That's fair. I'll go with saying that, in 2.7, server-side will be the default. No plan for removal of client-side.